src/globals.ts

```diff
@@ -41,21 +41,25 @@ export enum DroppedSpanReasons {
   INVOCATION_MAX_LATENCY_LIMIT = 'INVOCATION_MAX_LATENCY_LIMIT',
 }
 
+export type DroppedSpansData = {
+  drops: number;
+};
+
 export class SpansContainer {
-  private static spans: { [id: string]: BasicSpan } = {};
+  private static spans: { [id: string]: GenericSpan } = {};
   private static currentSpansSize: number = 0;
   private static totalSpans: number = 0;
-  private static droppedSpansReasons: { [reason: string]: number } = {};
+  private static droppedSpansReasons: { [reason: string]: DroppedSpansData } = {};
```

Comment:
why not just have a map of { string: number }
why is the type DroppedSpansData needed?